core/GlobalState.cc

```diff
@@ -835,6 +835,7 @@ void GlobalState::initEmpty() {
             Symbols::MAX_SYNTHETIC_TYPEARGUMENT_SYMBOLS);
 
     installIntrinsics();
+    computeLinearization();
```

Comment:
Motivation is also in this commit message:

```
This should be safe--if we ever call addMixin after this, we will reset
the linearization status on that class.

This allows us to call `verifyLinearizationComputed` in
`runIncrementalBestEffort` (because our test suite tests an edge case
where we run `runIncrementalBestEffort` with `--no-stdlib` so Resolver
truly has never been run before).
```